build/shared-runtime.version

```diff
@@ -1 +1 @@
-1.2.0-beta-001304-00
\ No newline at end of file
+1.1.1;release/1.1.0,2.0.0-beta-001747-00;master
```

Comment:
The problem comes from mismatch of package and runtime installation. The latest version of the [Microsoft.NETCore.App package](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App) on the dotnet-core feed is 2.0.0-beta-001753-00, but the [latest build of .NET Core CLI](https://github.com/dotnet/cli#installers-and-binaries) bundles a different **runtime**, `shared/Microsoft.NETCore.App/2.0.0-beta-001745-00`.

We could make the mirror step smart enough to ensure the package and the runtime match.

Or we can install both. But this will add an extra 45 MB download per build.